app/komponenter/confirmationPanel.tsx

```diff
@@ -0,0 +1,38 @@
+import { ConfirmationPanel } from '@navikt/ds-react';
+import { useState } from 'react';
+import { Button } from '@navikt/ds-react';
+import { SanityDokument } from '~/typer/sanity/sanity';
+import TekstBlokk from './tekstBlokk/tekstBlokk';
+
+interface BekreftelsePanelProp {
```

Comment:
```suggestion
interface Props {
```
Blir redundant med `BekreftelsePanel` forran `Prop`